src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -532,18 +526,24 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
       projectData.centroid = [-123.3656, 48.4284];
     }
 
-	// Set the text background colour for the shape file colour picker.
+		// Set the text background colour for the shape file colour picker.
     if (projectData.shapeFileColour) {
       this.shapeFileColour = projectData.shapeFileColour;
       this.shapeFileTextColour = this.colourIsBright(this.shapeFileColour) ? '#000000' : '#ffffff';
     }
 
+		// Choose which project phase list is shown: regular phases or forest phases.
     if (projectData.projectTypes) {
       this.projectTypes = projectData.projectTypes;
-    }
+			this.chosenPhases = projectData.projectTypes.find((type) => 'Forest Landscape Planning' === type.name)?.checked ? Constants.FOREST_PHASES : Constants.DEFAULT_PHASES;
+    } else {
+			this.chosenPhases = Constants.DEFAULT_PHASES;
+		}
 
-    const contactformEmailControls = Array.isArray(projectData.contactFormEmails) ? projectData.contactFormEmails.map(email => new FormControl(email)) : [];
+		// Remove the project phase value from the project if it doesn't exist in the chosen list of phases
+		projectData.projectPhase = this.chosenPhases.includes(projectData.projectPhase?.toString()) ? projectData.projectPhase : '';
 
+    const contactformEmailControls = Array.isArray(projectData.contactFormEmails) ? projectData.contactFormEmails.map(email => new FormControl(email)) : [];
```

Comment:
Just curious what this line is for. Was there a big that's being fixed? I appreciate that you're fixing little things along the way and just want to document it where we can

Reply:
Not sure why it's flagging this code as new. You can see in the existing codebase that it's on line 545: 
https://github.com/bcgov/landuseplanning-admin/blob/dev/src/app/projects/add-edit-project/add-edit-project.component.ts#L545